documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,59 @@ keywords:
 import create_vc_rs from  '!!raw-loader!./../../../examples/account/create_vc.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
 
+You can create a Verifiable Credential with the following properties:
```

Comment:
Even though we have an introduction page, it would be good to introduce creating a VC. Why would you want to create a VC, what does it mean to be an Issuer, who trusts your credentials? Doesn't have to be super long, but a bit of context is great.